src/lib/ens-helpers.ts

```diff
@@ -1,17 +1,16 @@
+import { bytesToPacket } from "@ensdomains/ensjs/utils";
 import { type Hex, concat, keccak256, namehash, toHex } from "viem";
 
-// TODO: pull from ens utils lib or something
 export const NAMEHASH_ZERO = namehash("");
 export const NAMEHASH_ETH = namehash("eth");
 
-// TODO: this should probably be a part of some ens util lib
+// TODO: this should probably be a part of some ens util lib but isn't exposed by viem or ensjs
 export const makeSubnodeNamehash = (node: Hex, label: Hex) => keccak256(concat([node, label]));
 
-// https://github.com/wevm/viem/blob/main/src/utils/ens/encodeLabelhash.ts
-export const encodeLabelhash = (hash: Hex): `[${string}]` => `[${hash.slice(2)}]`;
-
+// converts uint256-encoded nodes to hex
 export const tokenIdToLabel = (tokenId: bigint) => toHex(tokenId, { size: 32 });
 
+// TODO: this should be part of some more standard ens helper lib
```

Comment:
Please see a comment I shared above referencing the ENS library I was working on over the summer. This library includes a deeper solution to the issue this code is working to solve. Please feel welcome to add a task for me here too.